cmd/torusd/main.go

```diff
@@ -110,10 +113,34 @@ func configureServer(cmd *cobra.Command, args []string) {
 		fmt.Fprintf(os.Stderr, "error parsing read-cache-size: %s\n", err)
 		os.Exit(1)
 	}
-	size, err = humanize.ParseBytes(sizeStr)
-	if err != nil {
-		fmt.Fprintf(os.Stderr, "error parsing size: %s\n", err)
-		os.Exit(1)
+	if strings.Contains(sizeStr, "%") {
+		var stat syscall.Statfs_t
```

Comment:
@xiang90 splitting it makes sense, I'll do that. 

Yeah the current solution only would work on unix, not sure how wide spread torus is targeting for portability - the windows solution I saw was pretty shady, requiring linking the kernel32.dll.

I don't think we can use Sys() to get portability, looks like that was un-implemented for windows according to https://github.com/golang/go/issues/9611, I could be wrong on that though, I don't have a windows machine to test at this moment.

I'll look a the dependency, if we want Windows portability it may be worth it.